src/ShallowWrapper.js

```diff
@@ -350,8 +350,11 @@ export default class ShallowWrapper {
     const handler = this.prop(propFromEvent(event));
     if (handler) {
       withSetStateAllowed(() => {
-        // TODO(lmr): create/use synthetic events
         // TODO(lmr): emulate React's event propagation
+        if (!args.length && window) {
+          args.push(new window.Event(event));
```

Comment:
Don't most users have it available when running tests ?
I know its not requirement for shallow rendering, but are there many people that use only shallow rendering ?

Optionally I was thinking to add jsdom there but introducing this as dependency is probably not good idea.

Maybe just hardcode some event object with most used properties ?